packages/experiments/src/components/Chiclet/Chiclet.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import {
+  css
+} from '../../Utilities';
+import {
+  getClassNames
+} from './Chiclet.styles';
+import { IChicletProps, IChicletCardProps, IChicletStyles } from './Chiclet.types';
+import { mergeStyles } from '../../Styling';
+import { ChicletCard } from './ChicletCard';
+import { IButtonProps } from 'office-ui-fabric-react/lib/Button';
+
+export class Chiclet extends React.Component<IChicletProps, IChicletCardProps> {
+  private _classNames: IChicletStyles = {};
+
+  public render() {
+    const { styles: customStyles, chicletCardProps, size, actions, theme } = this.props;
```

Comment:
Can probably add `{ ... onClick = _onClick ... } = this.props;` and remove it from the manual DOM assignment below. I think we do it across the lib for functions that can be supplied via props including event handlers (tho this link isn't an event handler example): https://github.com/OfficeDev/office-ui-fabric-react/blob/fee79fb9e115dd282a4337418ac77ed4f1c6cad7/packages/office-ui-fabric-react/src/components/TextField/TextField.tsx#L150